test/invariants/fuzz/Protocol.t.sol

```diff
@@ -14,7 +14,7 @@ contract FuzzProtocol is EchidnaTest {
   BCoWFactory factory;
   BConst bconst;
   BMath bmath;
-  BNum_exposed bnum;
```

Comment:
why not BNum mock?

Reply:
they are not generated unless I I make the functions internal

Reply:
but we do have a MockBNum contract that externalizes all methods in `test/manual-smock/` dir

Reply:
addressing this on #179